scripts/ci/qemu-run.sh

```diff
@@ -199,6 +200,11 @@ if [[ -v DRIVE_PATH ]]; then
   QEMU_PARAMS="$QEMU_PARAMS -hda $DRIVE_PATH"
 fi
 
+# Add qcow2 drive
+if [[ -v QCOW2_DRIVE ]]; then
```

Comment:
@DaniilKl I think using `DRIVE_PATH` should be enough

Reply:
I have deleted this commit.